api/prisma/migrations/20250709223337_update_brand_translations/migration.sql

```diff
@@ -0,0 +1,17 @@
+-- Updates brand in translations for emails.
+
+UPDATE translations
+SET translations = jsonb_set(translations, '{header}', '{"logoUrl": "https://detroitmi.gov/themes/custom/detroitminew/logo.svg", "logoTitle": "Detroit Home Connect"}')
+WHERE language = 'en';
+
+UPDATE translations
+SET translations = jsonb_set(translations, '{footer,footer}', '"Detroit Home Connect"')
+WHERE language = 'en';
+
+UPDATE translations
+SET translations = jsonb_set(translations, '{forgotPassword,resetRequest}', '"A request to reset your Detroit Home Connect website password for %{appUrl} has recently been made."')
+WHERE language = 'en';
```

Comment:
Also, it looks like there are several additional translations we need to add here. I can send you a JSON of the translations in the deployed environment if that would help

Partner portal email: 
<img width="763" height="474" alt="Image" src="https://github.com/user-attachments/assets/83fb6e2f-5740-4ffe-b0bc-3adb0a14abec" />

Reply:
That'd be great @ludtkemorgan